### ISSUE TITLE
Rename branches to 6.x-1.x, 7.x-1.x, and 8.x-1.x

### ISSUE THREAD

**Issue by pietervanleuven** [mautic/mautic-drupal#36]
The branches should be renamed to reflect the Drupal.org structure (according to https://www.drupal.org/node/1015226):
- 6.x should be 6.x-1.x-dev,
- 7.x should be 7.x-1.x-dev,
- 8.x should be 8.x-1.x-dev

To facilitate further development, I think it's important we do this as quickly as possible, but after current pull-requests are reviewed, as renaming branches causes open pull-requests to be closed. (Can anyone confirm this?)

After this is done we should publish the current state of the module to Drupal.org as quickly as possible. This way everyone can see there's work being done in the dev-branches. If desired we can already make a release version of the current 7.x-1.x branch (that will be 7.x-1.0) and start getting feedback from the Drupal ecosystem.

So in short:
- [x] Merge or close current pull-requests (I think @jordan8037310 is working on that?)
- [x] Rename branches
- [ ] Push code to Drupal.org git-repo (and find an automated way to do this?)
- [ ] Promote current sandbox state on Drupal.org to full project (https://www.drupal.org/node/1068952)
- [ ] Create development release of 7.x-1.x branch.

Let me know if I can be of any assistance :) I have currently not enough permissions on Drupal.org to execute steps 3 and 4 myself.

**Comment by jordan8037310** [comment]
Merge and closed current pull-requests, sorry for the delay here. Was hoping for some community feedback but I'm just pressing on.

**Comment by jordan8037310** [comment]
The correct branch names are `6.x-1.x`, `7.x-1.x`, and `8.x-1.x`. They have been pushed up.

**Comment by jordan8037310** [comment]
@escopecz - I'm currently blocked on pushing this to Drupal.org. Doesn't seem like I have sufficient permissions to edit this sandbox: https://www.drupal.org/sandbox/mautic/2453263. Can you help?

**Comment by rteijeiro** [comment]
There is already a mautic module published in drupal.org (https://www.drupal.org/project/mautic). I'm contacting the module maintainer in order to merge the work we did here.

**Comment by jordan8037310** [comment]
Thanks—I previously reached out via the Slack group and did not receive a response...

**Comment by rteijeiro** [comment]
I'll try to have this sorted out as soon as possible. We have developed a few improvements to the module and it's a pity people are missing them due to a lack of response by the maintainer.

**Comment by RCheesley** [comment]
Closing this as the module is now managed on drupal.org
